webapp/components/info-banner.html

```diff
@@ -0,0 +1,56 @@
+<!--
+Copyright 2017 The WPT Dashboard Project. All rights reserved.
+Use of this source code is governed by a BSD-style license that can be
+found in the LICENSE file.
+-->
+
+<link rel="import" href="../bower_components/polymer/polymer-element.html">
+<link rel="import" href="../bower_components/paper-styles/color.html">
+
+<!--
+`<info-banner>` is a stateless component for displaying an information banner,
+of type info, warning, or error.
+-->
+<dom-module id="info-banner">
+  <template>
+    <style>
+      .banner {
+        padding: 0.5em;
+        background-color: var(--paper-blue-100);
+        margin-bottom: 1em;
+        margin-top: 2em;
+        border-left: solid 4px var(--paper-blue-300);
+      }
+      .banner small {
+        float: right;
+      }
+      .banner.error {
+        background-color: var(--paper-red-100);
+        border-left: solid 4px var(--paper-red-300);
+      }
+    </style>
+
+    <section class$="banner [[type]]">
+      <slot></slot>
```

Comment:
What's this for? Does it denote where content prior to content inside `<... slot="small">...</...>` will go?

Reply:
Correct - nameless is the default `slot`, which takes the (unslotted) child elements.
The other slot is named, so takes the element with `slot="small"`

https://www.polymer-project.org/2.0/docs/devguide/shadow-dom#shadow-dom-and-composition